docs/_static/css/custom-theme.css

```diff
@@ -77,7 +77,8 @@ img {
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-    max-width: 1200px;
+    max-width: 800px;
+    height: 100vh;
```

Comment:
This added height property is causing the grey background to cut off below the fold. See https://developers.gro-intelligence.com/musenlin-gro-patch-1/api.html and try scrolling